configs/ua-with-dev-dependencies.config

```diff
@@ -0,0 +1,2 @@
+npm.includeDevDependencies=true
```

Comment:
I would suggest naming this whitesource.config, for consistency with the expectations of LOCAL.
We can explain in a comment _when_ this file is used

Reply:
my original thought was here is the place to host shared ua configs and might have multiple files for sharing in future, so naming with more meaningful name is better. But I'm ok with `whitesource.conf` at the moment.